db/migrate/20241014031821_add_deleted_at_to_course_groups.rb

```diff
@@ -0,0 +1,6 @@
+class AddDeletedAtToCourseGroups < ActiveRecord::Migration[7.0]
+  def change
+    add_column :course_groups, :deleted_at, :datetime
```

Comment:
Why does Course Groups need soft-delete for course-user deletion?

Reply:
From this [line](https://github.com/Coursemology/coursemology2/blob/f84dfd0b3faface9645177ccafd93ca80e694187/app/models/course_user.rb#L58) I am assuming need delete it if delete course-user.

Reply:
That's Course::GroupUser not Course::Group right?

Reply:
Yes, thats correct. I have reverted this change, its outdated now.